proto/autoops/service.proto

```diff
@@ -160,6 +161,67 @@ message ListWebhooksResponse {
   int64 total_count = 3;
 }
 
+message CreateProgressiveRolloutRequest {
+  string environment_namespace = 1;
+  CreateProgressiveRolloutCommand command = 2;
+}
+
+message CreateProgressiveRolloutResponse {}
+
+message GetProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+}
+
+message GetProgressiveRolloutResponse {
+  ProgressiveRollout progressive_rollout = 1;
+}
+
+message DeleteProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+  DeleteProgressiveRolloutCommand command = 3;
```

Comment:
question:

What is `DeleteProgressiveRolleoutCommand` needed for?

Reply:
This is used in https://github.com/bucketeer-io/bucketeer/blob/main/pkg/autoops/command/auto_ops_rule.go#L50.

For example DeleteFeatureCommand is defined in https://github.com/bucketeer-io/bucketeer/blob/main/proto/feature/command.proto#L50.